AUTHORS

```diff
@@ -1,6 +1,7 @@
 AUTHORS
 -------
 
-Corey Nolet - Tetra Concepts LLC
-Keith Turner - Peterson Technologies
-Mike Walch - Peterson Technologies
+Corey Nolet (Tetra Concepts LLC)
+Keith Turner (Peterson Technologies)
+Mike Walch (Peterson Technologies)
+Christopher Tubbs
```

Comment:
I assumed it was sorted by "earliest edit (to the file)" :)
I actually think it would be best to delete this file, and just keep the information in the POM. The AUTHORS file actually seems a bit redundant to me.